content/blog/2019/08/2019-08-23-introduce-react-plugin-template.adoc

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: "Introduce React Plugin Template"
+tags:
+- react
+- plugins
+- template
+- gsoc
+- gsoc2019
+author: jackshen
+---
+The template's main repo is at 
+link:https://github.com/jenkinsci/react-plugin-template[React Plugin Template]
+
+This template is part of the project link:https://summerofcode.withgoogle.com/projects/#6112735123734528[Working Hours UI Improvement] during
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2019], which improve the UI of `Working Hours Plugin` using this pattern to develop Jenkins plugin with React. The `Working Hours Plugin` repository could be found at link:https://github.com/jenkinsci/working-hours-plugin[Working Hours Plugin].
```

Comment:
Please change the following:
`The `Working Hours Plugin` repository could` to `The `Working Hours Plugin` repository can`